package.json

```diff
@@ -97,12 +101,13 @@
     "rollup-plugin-filesize": "6.1.1",
     "rollup-plugin-includepaths": "0.2.3",
     "rollup-plugin-json": "4.0.0",
+    "rollup-plugin-node-globals": "1.4.0",
     "rollup-plugin-node-resolve": "5.2.0",
     "rollup-plugin-replace": "2.2.0",
-    "rollup-plugin-node-globals": "1.4.0",
     "rollup-plugin-uglify": "6.0.2",
     "rollup-watch": "4.3.1",
-    "streamtest": "1.2.4"
+    "streamtest": "1.2.4",
+    "typescript": "^3.5.3"
```

Comment:
I see all other deps use pinned versions. We should probably do the same for the newly installed ones.

E.g. 
~~"typescript": "^3.5.3"~~
"typescript": "3.5.3"